lib/sinon/collection.js

```diff
@@ -11,12 +11,11 @@
 var sinonSpy = require("./spy");
 var sinonStub = require("./stub");
 var sinonMock = require("./mock");
-var walk = require("./util/core/walk");
-var getPropertyDescriptor = require("./util/core/get-property-descriptor");
-var valueToString = require("./util/core/value-to-string");
+var failOnFalsyObject = require("./fail-on-falsy-object");
```

Comment:
how about `throwOnFalsyObject` - as this tells you explicitly what this will do. `fail` could mean many different things.